libbeat/docs/newbeat.asciidoc

```diff
@@ -52,7 +52,14 @@ go get github.com/elastic/beats
 
 When you run the command, all source files are downloaded to the
 `$GOPATH/src/github.com/elastic/beats` path. You can ignore the "no buildable Go source files" message because
-you will build the source later.
+you will build the source later. By default `go get`  fetches the master branch. To build your beat
+on a specific version of libbeat, check out the specific branch (5.1 in the example below):
+
+[source,bash]
+----
+cd $GOPATH/src/github.com/elastic/beats
+git checkout 5.1
```

Comment:
we could you {doc_branch} here but that will not work in the docs above which are also shown on Github. We have to find some simplification here in the future.